EpsilonCCS/Src/MpptRtr.c

```diff
@@ -0,0 +1,53 @@
+#include "cmsis_os.h"
+
+#include "MpptRtr.h"
+
+void mpptRtrTask(void const* arg)
+{
+    uint32_t prevWakeTime = osKernelSysTick();
+    uint32_t channel = 0;
+
+    for (;;)
+    {
+        osDelayUntil(&prevWakeTime, MPPT_HEARTBEAT_CAN_FREQ);
+        // Allocate CAN Message, deallocated by sender "sendCanTask()"
+        MpptCanMsg* msg = (MpptCanMsg*)osPoolAlloc(canTxPool);
+        // Populate CAN Message
+        msg->stdId = MPPT_STDID;
+        msg->channel = channel;
+
+        // Update Channel
+        if (channel = 3)
```

Comment:
Sorry man, but change it back it looked better before haha (I was wrong thinking it would overflow before).